From 3251456eee97f804af07a833a03cca5b63845c66 Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Wed, 1 Dec 2021 05:17:51 -0300
Subject: [PATCH 3/3] Use clearlinux CFLAGS during build

---
 Configurations/10-main.conf | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Configurations/10-main.conf b/Configurations/10-main.conf
index 61c6689..f7b1657 100644
--- a/Configurations/10-main.conf
+++ b/Configurations/10-main.conf
@@ -657,6 +657,8 @@ my %targets = (
     },
     "linux-generic64" => {
         inherit_from     => [ "linux-generic32" ],
+        CFLAGS           => add($ENV{'CFLAGS'}),
+        ex_libs          => "$ENV{'EX_LIBS'}",
         bn_ops           => "SIXTY_FOUR_BIT_LONG RC4_CHAR",
     },
 
@@ -767,6 +769,8 @@ my %targets = (
     "linux-elf" => {
         inherit_from     => [ "linux-generic32", asm("x86_elf_asm") ],
         CFLAGS           => add(picker(release => "-fomit-frame-pointer")),
+        CFLAGS           => add($ENV{'CFLAGS'}),
+        ex_libs          => "$ENV{'EX_LIBS'}",
         lib_cppflags     => add("-DL_ENDIAN"),
         bn_ops           => "BN_LLONG",
     },
-- 
2.34.0

